tests/test_util.cuh

```diff
@@ -121,6 +150,57 @@ inline uint64_t Murmur3HashHost(const uint64_t& key) {
   return k;
 }
 
+template <class K, class M, class V, size_t DIM = 16>
+void create_continuous_keys(K* h_keys, M* h_metas, V* h_vectors, int KEY_NUM,
```

Comment:
Should remove the same func from `merlin_hashtable_test.cc.cu` and please notice the `DIM`.